Puzzlepart/prosjektportalen-program
# First draft of splitting queries

### Your checklist for this pull request

- [x] Make sure you are requesting to **pull a feature/bugfix/issue branch** (right side).

### Description

Splits queries in @Common 

### How to test

Several projects required. Above 40 most likely.
Add all to Program.
Navigate to Program Portfolio
View Network tab if two post queries are made to the REST API.
If successful, all projects should be viewable.

Repeat this for BenefitsOverview, ProgramDeliveries, RiskOverview, ResourceAllocation

### Relevant issues (if applicable)

Closes #51
